manifests/endpoint.pp

```diff
@@ -41,13 +41,15 @@
     if $manage_ssh_rules {
         # Disable SSH rate limiting on presumed-to-be-safe networks
         if $unthrottled_networks {
-            firewall { '001 ipv4 accept unthrottled ssh':
-                protocol => 'iptables',
-                source   => $unthrottled_networks,
-                chain    => 'INPUT',
-                proto    => 'tcp',
-                dport    => 22,
-                jump     => 'accept',
+            $unthrottled_networks.each |$nw| {
+                firewall { '001 ipv4 accept unthrottled ssh':
```

Comment:
The title needs to have ${nw} in it so that resource titles are unique in case of multiple $unthrottled_networks. Otherwise it goes boom.

Reply:
Done